### ISSUE TITLE
Clean up webmaker logo folder on Drive

### ISSUE THREAD

**Issue by cassiemc** [MozillaFoundation/Design#83]
The Webmaker folder on drive, in particular the Webmaker-brand-2015 logo, is hard to parse. We should make this easier to go in and find the final logos you need. It's hard to tell what is old and new. It actually seems to me like the final logo files aren't really there – or if they are, they're not well-documented.

@lovegushwa can you own this and get it done in the next couple of days? There is a Moco bug that needs the new Webmaker assets and I'd like to be able to find them quickly on this drive. Here's the moco bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1175199

cc @xmatthewx @vazquez

**Comment by lovegushwa** [comment]
Applied the new file naming conventions found here:
https://docs.google.com/a/mozilla.com/document/d/1AZ_TrLkdWVnRT_fAxW-kFT3o2tDgMlg_f65AKKcrtJY/edit?usp=sharing

You will find the cleaned up logo files folder here:
https://drive.google.com/a/mozilla.com/folderview?id=0B2qu3ExKWIlvaGZWTUhRWng3RzQ&usp=sharing

**Comment by cassiemc** [comment]
thank you @lovegushwa!
